app/controllers/service_requests_controller.rb

```diff
@@ -437,9 +437,9 @@ def remove_service
     to_delete = @service_request.sub_service_requests.map(&:organization_id) - @service_request.service_list.keys
     to_delete.each do |org_id|
       ssr = @service_request.sub_service_requests.find_by_organization_id(org_id)
-      if !['first_draft', 'draft'].include?(@service_request.status) and !@service_request.submitted_at.nil? and @service_request.submitted_at > ssr.created_at
+      if @service_request.line_items.count.zero? && !ssr.submitted_at.nil?
```

Comment:
Yeah!  Trigger: When a user goes back to a protocol through "Edit" or "Modify Request" button, and delete all the services in an SSR from his/her shopping cart, this email should be send out to the service provider(s) when the last service on a SSR has been deleted from the SPARCRequest shopping cart.  Before it would send out on each removed service, but Wenjun wanted that changed.  Here's the link to the story:  https://www.pivotaltracker.com/story/show/130746829.  If you look at Wenjun's last comment, she doesn't want the logic to rely on the status anymore, only submitted_at.  Does that make sense?